panther_hardware_interfaces/src/phidgets_spatial_parameters.yaml

```diff
@@ -0,0 +1,144 @@
+# https://github.com/ros-drivers/phidgets_drivers/blob/humble/phidgets_spatial/README.md
```

Comment:
Is it a common practice to store .yaml files in the src directory? Would it be better to create a config or params directory for them instead?

Reply:
Okay, so I have no strong opinion on this matter.